.github/workflows/trivy.yml

```diff
@@ -84,11 +84,20 @@ jobs:
           - agentplane-azure-vault
           - agentplane-hashicorp
     steps:
+
+      - name: Check github repository and set docker repo
+        id: set-docker-repo
+        run: |
+          [ "${{ github.repository }}" == "eclipse-tractusx/knowledge-agents-edc" ] && echo "REPO=tractusx" >> $GITHUB_OUTPUT
+          [ "${{ github.repository }}" == "eclipse-tractusx/knowledge-agents-edc" ] && echo "REPO=ghcr.io/catenax-ng/tx-knowledge-agents-edc" >> $GITHUB_OUTPUT
```

Comment:
Same here, workflow shall not care about running in a fork.

Reply:
Sorry, order of comment is different than in PR review. Please refer to to the other comment.

Reply:
thanks for the heads up, I think I found a way that now really focusses on eclipse-tractusx and does not reference any specific fork. Thanks for revisiting the PR ...